apps/web/src/pages/Pools/EditPoolPage/components/GeneralQuestions.tsx

```diff
@@ -24,37 +24,37 @@ const MAX_SCREENING_QUESTIONS = 3;
 const TEXT_AREA_ROWS = 3;
```

Comment:
Can rename MAX_SCREENING_QUESTIONS

Reply:
Oops, missed that. Got it in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9070/commits/94c40af5cf303cb21ba4a5faa2c196d2c5f411db